tests/PHPStan/Analyser/data/bug-7776.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types = 1);
+
+namespace Bug7776;
+
+use function PHPStan\Testing\assertType;
+
+/**
+ * @param array{page?: int, search?: string} $settings
+ */
+function test(array $settings = []): bool {
+	$copy = [...$settings];
+	assertType('array{page?: int, search?: string}', $copy);
+	assertType('array{page?: int, search?: string}', $settings);
+	return isset($copy['search']);
+}
+
+test();
```

Comment:
This line should be dropped as it is not necessary and executes code while running tests

Reply:
thx!